client/src/components/pages/RobotFile/RobotFile.jsx

```diff
@@ -1,21 +1,79 @@
-import React from 'react';
-import { Layout, Typography } from 'antd';
+import React, { useState, useEffect } from 'react';
+import { Layout, Button, Space, Row, Col } from 'antd';
+import Editor from 'react-simple-code-editor';
+import { highlight, languages } from 'prismjs/components/prism-core';
 import HeaderNavbar from '../../content/HeaderNavbar/HeaderNavbar';
-
-const { Title } = Typography;
+import 'prismjs/components/prism-robotframework';
+import getParsedRobotFile from '../../../api/ssot';
+import 'prismjs/themes/prism.css';
+import styles from './RobotFile.module.css';
 
 /**
  * @description View of the robot file
  * @category Client
  * @component
  */
-const RobotFile = () => (
-  <Layout>
-    <HeaderNavbar selectedKey={3} />
-    <Title style={{ paddingLeft: '2rem', paddingTop: '2rem' }}>
-      In future versions, the Robot Framework code can also be edited directly here.
-    </Title>
-  </Layout>
-);
+const RobotFile = () => {
+  const [code, setCode] = useState(
+    'Please wait, your robot file is being loaded.'
+  );
+
+  /**
+   * @description Equivalent to ComponentDidMount in class based components
+   */
+  useEffect(() => {
+    const robotId = JSON.parse(sessionStorage.getItem('robotId'));
+    getParsedRobotFile(robotId)
+      .then((response) => response.text())
+      .then((robotCode) => {
+        setCode(robotCode);
+      });
+  }, []);
+
+  /**
+   * @description Gets called when the the button is pressed to save to the cloud.
+   * This function will retrieve the code from the editor, parse it to a ssot and write the
+   * resulting ssot into the sessionStorage.
+   */
+  const onSaveToCloud = async () => {
+    const robotId = JSON.parse(sessionStorage.getItem('robotId'));
+    console.log(robotId);
+    console.log(code);
```

Comment:
maybe to fix the linter issues:
```suggestion
    console.warn(robotId,code);
```

Reply:
But it is not really a warning, so I wouldn't want to use that.